tests/client.spec.js

```diff
@@ -248,6 +248,24 @@ describe("Client", () => {
         });
       });
 
+      // Fixture: disjunctive_license_returning_no_facets
+      // Fixture: search_filter_and_multi_facet_returning_no_facets
+      it("will handle not having `facets` in the backend response without throwing", async () => {
+        const result = await client.search("dog", {
+          ...config,
+          filters: {
```

Comment:
The change looks good but the testcase is incorrect. What you want in the testcase is to validate that the request doesn't have the presence of facets object. 

I think you should see this behavior (without the fix) with the following configuration:

 ```typescript
const result = await client.search("dog", {
          ...config,
          filters: {
            license: "BSD"
          },
          facets: {
            dependencies: [{ type: "value", size: 3 }]
          },
          disjunctiveFacets: ["license"]
        });
```

The other way is to filter disjunctiveFacets array that are only present in facets objects too.

Reply:
I don't think that's correct; I don't want to validate the **request** not having the presence of the facet object. It's the **response** what I really care about. That's why I do the assertion down below.  This is really similar to the original request that produced the issue for me if you look back to the original slack thread.

Reply:
The behavior doesn't reproduce by modifying the request as you suggested. It produces a different error: 

![image](https://user-images.githubusercontent.com/55319644/231151974-15dfa674-aec7-4128-a637-69753e864926.png)

Because `Facet-Only` request goes out to fetch the disjunctive "license" tag
![image](https://user-images.githubusercontent.com/55319644/231152424-62accd53-481a-4fe2-b9fb-a36c254bed6e.png)

Reply:
@joemcelroy any updates?